website/docs/r/cluster.html.markdown

```diff
@@ -275,6 +308,21 @@ Physical location of the region.
 
     If you do not specify this option, no analytics nodes are deployed to the region.
 
+### Advanced Configuration Options
+
```

Comment:
Add beneath the header but before the options:  Prior to setting these options please ensure you read https://docs.atlas.mongodb.com/cluster-config/additional-options/.

Reply:
okay, thanks @themantissa 👍